src/net_processing.cpp

```diff
@@ -1723,6 +1729,9 @@ void PeerManagerImpl::BlockConnected(const std::shared_ptr<const CBlock>& pblock
             m_txrequest.ForgetTxHash(ptx->GetWitnessHash());
         }
     }
+    if (m_chainman.ActiveChainstate().IsInitialBlockDownload()) {
+        m_block_stalling_timeout = std::max(m_block_stalling_timeout.load() / 2, DEFAULT_BLOCK_STALLING_TIMEOUT);
```

Comment:
The decrease does not log anything, but increase does. For symmetry and more log clarity, is it better to log both operations?

Reply:
Done.